front-end-web/src/Components/Pages/ProfilePage.tsx

```diff
@@ -0,0 +1,184 @@
+import {
+  Button,
+  Typography,
+  TextField,
+  Dialog,
+  DialogContentText,
+  DialogContent,
+  DialogActions,
+  DialogTitle
+} from "@material-ui/core";
+import React, { Fragment } from "react";
+import firebase from "firebase";
+import firebaseApp from "../../scripts/firebaseConfig";
+import { NotificationMessage } from "../../@Types";
+declare interface ProfileProps {
+  openPage: boolean;
+  handleToggleProfile: () => void;
+
+  handleNewNotification: (NewNotification: NotificationMessage) => void;
+}
+
+/**
+ * TODO: Add Documentation
+ */
+
+let newEmail: string = "";
+let newPassword: string = "";
+let newPasswordCheck: string = "";
+let newName: string = "";
+let currentName: string = "";
+let currentEmail: string = "";
+
+firebaseApp.auth();
```

Comment:
You shouldn't need this just hanging out, it should already be initialized for you.